pyproject.toml

```diff
@@ -8,7 +8,7 @@ authors = [
 ]
 requires-python = ">=3.12"
 dependencies = [
-    "repoauditor>=0.1.22",
+    "repoauditor>=0.1.24,<1.0.0",
```

Comment:
Any reason to restrict the version to \< 1.0? There are different schools of thought on this and I am wondering why you decided to go this route.